src/components/leaderboard/LeaderboardRow/index.tsx

```diff
@@ -0,0 +1,32 @@
+import Image from 'next/image';
+import styles from './style.module.scss';
+
+interface LeaderboardRowProps {
+  position: number;
+  rank: string;
+  name: string;
+  points: number;
+  image: string;
+}
+
+const LeaderboardRow = ({ position, rank, name, points, image }: LeaderboardRowProps) => {
+  return (
+    <div className={styles.row} data-style={position % 2 === 0 ? 'even' : 'odd'}>
+      <span className={styles.position}>{position}</span>
+      <Image
+        src={image}
```

Comment:
Code Suggestion: Since we're working with the next/image component, it might be worth considering leveraging its auto compression capabilities on the images of these leaderboard rows since every KB counts when we're rendering so many of them on one page. Plus, these images are small so the compression will be less noticeable, yay!  We can do this by using the `quality={}` prop. I tested this on m computer and found that setting the quality to 10/100 dropped the rendered file size by about half from 746kb to 300ish.